specs/eip4844/beacon-chain.md

```diff
@@ -97,6 +104,7 @@ class BeaconBlockBody(Container):
     sync_aggregate: SyncAggregate
     # Execution
     execution_payload: ExecutionPayload 
+    bls_to_execution_changes: List[SignedBLSToExecutionChange, MAX_BLS_TO_EXECUTION_CHANGES]
```

Comment:
Otherwise we're gonna have to create new objects that will never exist (an eip4844 block minus withdrawals)

Reply:
yeah we wanna retain the fields to make it easy for client devs to branch off their existing withdrawals implementation. I expect little to no bugs/changes to that implementation and specification.

Only thing we wanna avoid is triggering state transition logic for withdrawals since that's where they could most likely be different behaviors in client implementations.